### ISSUE TITLE
Discontinued?

### ISSUE THREAD

**Issue by KiryuKazumaLikeadragon** [mirinsoft/CloneApp#33]
Last update is 2020 and i see many issue remain unresolved, is this app no longer maintaned?

**Comment by sheikhrehman** [comment]
same question

**Comment by Technetium1** [comment]
No, it's not maintained. The last commit is 4 years ago. The final release also states this.
https://github.com/builtbybel/CloneApp/releases/tag/2.15.606
> CloneApp is no longer actively being developed
